plugins/modules/copr.py

```diff
@@ -255,6 +259,8 @@ def _enable_repo(self, repo_filename_path, repo_content=None):
         """
         if not repo_content:
             repo_content = self._download_repo_info()
+        if self.ansible_module.params.get("includepkgs"):
+            repo_content += "\nincludepkgs={}\n".format(self.ansible_module.params['includepkgs'])
```

Comment:
Why do you need to put `\n` on both sides? Does the downloaded content not end in `\n`?

Reply:
Because want  includepkgs parameter is properly formatted within the repository content.
These newlines are added to prevent any formatting issues, ensuring that the includepkgs directive is always placed on its own line, regardless of how the original content ends.
added at end because content from "_download_repo_info()"  might not necessarily end with a newline and  appending would  create an invalid configuration due to merging of lines .
To avoid this  example :
```
[repo]
name=Example Repo
baseurl=http://example.com/repo
enabled=1
gpgcheck=1includepkgs=package1,package2
```

Reply:
You could remove trailing newlines first: `repo_content = repo_content.rstrip('\n') + '\nincludepkgs=...'`. That would make sure that `includepkgs` is always in its own line without having empty lines between the previous content and `includepkgs`.

Reply:
thanks for the advice updated the repo_content and also was kind of stuck on sanity test and this help 
thank you

Reply:
if any other change needed please provide update